static/app/types/views.tsx

```diff
@@ -0,0 +1,6 @@
+export type View = {
+  name: string;
+  query: string;
+  querySort: 'date' | 'new' | 'trend' | 'freq' | 'user' | 'inbox';
```

Comment:
Could use use the existing enum `IssueSortOptions`?

Reply:
yup! did not see that one my bad